src/services/attendees/attendee-schema.ts

```diff
@@ -7,9 +7,11 @@ const AttendeeValidator = z.object({
     name: z.string(),
     email: z.string().email(),
     events: z.array(z.string()),
-    dietary_restrictions: z.string(),
+    dietary_restrictions: z.string().array(),
+    allergies: z.string().array(),
     priority_expiry: z.date().nullable().optional(),
     points: z.number().min(0).default(0),
+    checkin: z.boolean().array().default([false, false, false, false, false]),
```

Comment:
change this to hasPriority, should be a dictionary of days (or at least extend some interface for us to check if priority is given on a certain day)